app/components/ChartDataFormatter/index.js

```diff
@@ -20,7 +20,9 @@ class ChartDataFormatter extends Component {
     this._applyFormatting = this._applyFormatting.bind(this);
     this.formatGuidelines = [
       'Formatting is applied to y-axis and tooltip labels.',
-      'Learn more about D3 formatting syntax <a href="https://github.com/d3/d3-format/blob/master/README.md">here</a>.',
+      'Learn more about D3 formatting syntax <a '
```

Comment:
If you update your eslintrc you don't need to do this: https://github.com/alleyinteractive/ux-et-developer-handbook/blob/master/content/code-share/.eslintrc#L32-L37